protobufs/livekit_sip.proto

```diff
@@ -424,14 +424,22 @@ message CreateSIPParticipantRequest {
   // If true, a random value for identity will be used and numbers will be omitted from attributes.
   bool hide_phone_number = 10;
 
+  // These headers are sent as-is and may help identify this call as coming from LiveKit for the other SIP endpoint.
```

Comment:
I think we should just keep headers, and remove the mappings fields. the mappings are a level of indirection away and it's a bit difficult to understand.

Reply:
Makes sense! Made a new commit based on this comment